packages/turf-truncate/index.js

```diff
@@ -0,0 +1,55 @@
+const deepSlice = require('deep-slice');
+
+/**
+ * Takes a GeoJSON Feature or FeatureCollection and truncates the precision of the geometry.
+ *
+ * @name truncate
+ * @param {Feature|FeatureCollection} layer any GeoJSON Feature or FeatureCollection
+ * @param {number} [precision=6] coordinate decimal precision
+ * @param {number} [coordinates] maximum number of coordinates
+ * @returns {Feature|FeatureCollection} layer with truncated geometry
+ * @example
+ * var point = {
+ *     "type": "Feature",
+ *     "geometry": {
+ *         "type": "Point",
+ *         "coordinates": [
+ *             70.46923055566859,
+ *             58.11088890802906
+ *         ]
+ *     },
+ *     "properties": {}
+ * };
+ * var pointTrunc = turf.truncate(point, 6);
+ * //= pointTrunc
+ */
+module.exports = function (layer, precision = 6, coordinates) {
```

Comment:
similarly, default parameter values are an ES6 feature and would increase Turf's support requirement.